Makefile.inc

```diff
@@ -36,6 +37,21 @@ $(shell echo "rustc --version  = $(shell rustc --version)" 1>&2)
 $(shell echo "rustup --version = $(shell rustup --version)" 1>&2)
 $(shell echo "**********" 1>&2)
 
+# The default target: build the board.
+mainboard: $(IMAGE)
```

Comment:
Hmmm. We're include this Makefile.inc file for every crate, but most of these variables only apply to (and are set for) mainboards. So these targets will do weird stuff on all the non-mainboard makefiles.

Reply:
we can fix that, I think, on the next pass. Good point.